modules/jsdom/src/polyfills/events/monitor.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2020, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {map, publish, refCount} from 'rxjs/operators';
+
+import {glfw, GLFW, GLFWmonitor} from '../../../../glfw/src/glfw';
+import {Monitor} from '../../../../glfw/src/monitor';
```

Comment:
These paths won't be valid when we run the compiled code. We'll need to export these types from the glfw module in `index.ts`